types/sinon/sinon-tests.ts

```diff
@@ -443,6 +443,10 @@ function testSpy() {
     arr = spy.exceptions;
     arr = spy.returnValues;
 
+    let arg: sinon.SinonSpyCall;
```

Comment:
whoops i totally misread this the first time. we need to be checking `spy.firstArg` don't we rather than `firstCall`.

i guess it has to be `any` after all in that case

Reply:
I totally bungled that... I got `firstCall` and `firstArg` confused...

Reply:
updated now